cmd/gobgpd/main.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	"github.com/coreos/go-systemd/daemon"
 	"github.com/golang/protobuf/ptypes/any"
-	"github.com/jessevdk/go-flags"
```

Comment:
why?

Reply:
Oops , I'm sorry, I didn't noticed that. This caused by my vscode plugin.
I fixed it.